4.4 Calc.py

```diff
@@ -0,0 +1,37 @@
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+def sum (a, b):
+    logging.debug("Ten program dodał do siebie liczbę %i oraz %i" % (a, b))
+    print ("Wynik działania to: %i" % (a+b))
+def difference(a, b):
+    logging.debug("Ten program odjoł do siebie liczbę %i oraz %i" % (a, b))
+    print ("Wynik działania to: %i" % (a-b))
+def multiplication(a, b):
+    print ("Wynik działania to: %i" % (a*b))
+    logging.debug("Ten program pomnożył dwie liczby %i oraz %i" % (a, b))
+def division(a, b):
+    logging.debug("Ten program podzielił przez siebie liczby %i oraz %i" % (a, b))
+    if b == 0:
+        print("Nie można wykonanć działania dzielenie przez 0 niemożliwe")
+    else:
+        print ("Wynik działania to: %i" % (a/3b))
+
+
+
+if __name__ == "__main__":
+    calc_function = input("Podaj działanie, posługując się odpowiednią liczbą: 1 Dodawanie, 2 Odejmowanie, 3 Mnożenie, 4 Dzielenie: ")
```

Comment:
Ze spokojem można byłoby to przenieść do funckji: 'cli_interface()' i ją wywołać w main()
Przypisanie liczb do poszczególnych funkcji powinno odbyć się w jakieś strukturze: np słownik a komunikat generowany automatycznie. 
Staramy się, aby pojedyncza modyfikacja w treści zadania, nie wiązała się z wieloma modyfikacjami istniejących linii w kodzie (kod odporny na modyfikacje, łatwy do rozszerzenia) - tymczasem chowanie logiki w stringach sprawia, że ktoś będzie musiał mocno wgryźć się w kod, aby zauważyć, gdzie jeszcze musi na przykład dodać obsługę kolejnego działania matematycznego.